src/main/java/com/google/firebase/auth/internal/ListTenantsResponse.java

```diff
@@ -17,20 +17,24 @@
 package com.google.firebase.auth.internal;
 
 import com.google.api.client.util.Key;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
+import com.google.firebase.auth.ListTenantsPage;
 import com.google.firebase.auth.Tenant;
 import java.util.List;
 
 /**
  * JSON data binding for ListTenantsResponse messages sent by Google identity toolkit service.
  */
-public class ListTenantsResponse {
+public final class ListTenantsResponse {
 
   @Key("tenants")
-  private List<Tenant> tenants;
+  private List<Tenant> tenants = ImmutableList.of(); // Default to empty list.
```

Comment:
We don't use defaults in `DownloadAccountResponse`. Why were they necessary here?

Reply:
The reason we didn't have to use them in DownloadAccountResponse is because of what we do in ListUsersPage.DefaultUserSource.fetch() with the ListUsersResult class as an intermediate storage class. We handle the null case and empty list case there.

But in ListTenantsPage, we decided not to include ListTenantsResult.

Having these defaults set in ListTenantsResponse does feel clunky to me though. Perhaps I should re-add the ListTenantsResult class for consistency?

Reply:
Perhaps handle them in the getters?

```
public List<Tenant> getTenants() {
  if (tenants != null) {
    return tenants;
  }

  return ImmutableList.of();
}
```